pkg/lint/load.go

```diff
@@ -138,6 +163,7 @@ func separateNotCompilingPackages(lintCtx *linter.Context) {
 	}
 }
 
+//nolint:gocyclo
```

Comment:
comment on exported function LoadContext should be of the form "LoadContext ..."

Reply:
this issue reportin was fixed in current pull request